spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AuditRecordControllerTests.java

```diff
@@ -127,6 +129,9 @@ public void setupMocks() throws Exception {
 		mockMvc.perform(post("/streams/definitions/").param("name", "myStream2").param("definition", "time | log")
 				.accept(MediaType.APPLICATION_JSON)).andExpect(status().isCreated());
 
+		Awaitility.await().atMost(Duration.ofMillis(30000)).until(() -> auditRecordRepository.count() == 7);
```

Comment:
Oh my,  a 🪄 #️⃣ ... where does the 7 come from?

Reply:
Great question.  I added a comment and a descriptive constant explaining where the 🪄 #️⃣ is derived. Basically its 4 app registration audits and 3 stream create audits.   @onobc Great Catch!